md.py

```diff
@@ -49,13 +49,15 @@ def _add_action(self, action):
         return super()._add_action(action)
 
 class MDArgumentParser(EnvironmentArgumentParser):
-    def __init__(self):
+    def __init__(self, md_nargs=1):
```

Comment:
I don't like this much at all. I think it'd be better to just replace these arguments in the perf script like this:

```python3
parser = md.MDArgumentParser(conflict_handler='resolve')
parser.add_argument("-c", "--chunk-size", default=[64 << 10],
                    type=parser._suffix_parse, nargs="+",
                    help="md chunk size (specify multiple arguments to sweep)")

args = parser.parse_args()
```

(Note the conflict_handler in the parser instantiation)

But maybe add a commit to remove the single underscore on suffix_parse that I added.

Reply:
Yeah, I didn't like it much either, the conflict_handler is way nicer